routes/api.js

```diff
@@ -158,6 +159,66 @@ router.patch('/user/badge/update', authenticationHelpers.isAuthOrRedirect, (req,
     }
 });
 
+router.post('/user/activity/level', authenticationHelpers.isAuthOrRedirect, (req, res) => {
+    if (!req.user.email) {
+        return res.status(400).json({error: errorCode.PCE021.message, code: errorCode.PCE021.code});
+    }
+    localUser.find({where: {
+        email: req.user.email
+    }}, {raw: true})
+        .then(data => {
+            if (!data) {
+                return res.status(200).json({info: errorCode.PCE022.message, code: errorCode.PCE022.code});
```

Comment:
error code..

Reply:
@shreyans29 error code seems correct to me?

Reply:
He means to say that res.status should not be 200 since its an error. It should be 402.

Reply:
Okay! Got it.